mesheryctl/cmd/root.go

```diff
@@ -107,6 +105,7 @@ func init() {
 	// when this action is called directly.
 	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
 	rootCmd.Flags().BoolP("version", "v", false, "Version flag")
+	rootCmd.Flags().BoolP("version", "version", false, "Version flag")
```

Comment:
@ashiscs did you tried it by creating a local binary, and check it?

Reply:
@ashiscs so do one this this changes are not required, revert these changes and resolve the help msg that is printed when we use the `-v` or  `--version` flag

Reply:
OK